templates/zerver/help/send-a-stream-message.md

```diff
@@ -1,31 +1,35 @@
 ## Sending a new stream message
 
-![New stream message](/static/images/help/new-stream-message.png)
+This guide is to teach you how to send a message to a new stream.
```

Comment:
"This guide is to teach you" is improper English. Try "You can follow the following steps to send a new stream message"